receiver/podmanreceiver/README.md

```diff
@@ -1,12 +1,15 @@
 # Podman Stats Receiver
+| Status                   |            |
+| ------------------------ |------------|
+| Stability                | [alpha]    |
```

Comment:
This component should be marked [unmaintained](https://github.com/open-telemetry/opentelemetry-collector#unmaintained).